getting-started.md

```diff
@@ -132,11 +132,16 @@ sure that the rekor entry is created for it.
 kubectl -n ctlog-system get secrets ctlog-public-key -oyaml | sed 's/namespace: .*/namespace: default/' | kubectl apply -f -
 ```
 
-2) Create the three test jobs (checktree, sign-job, and verify-job)  using this
+2) Get fulcio-secret and add to default namespace
```

Comment:
can you add a comment on how these secrets are used by the test?

Reply:
sorry, saw that you and @priyawadhwa  had approved, but didn't see this comment. I'll do in a followup :(